nifi-testharness/src/test/resources/sample_technology_rss.xml

```diff
@@ -0,0 +1,28 @@
+
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this xml document looks like a real http response.  We cannot use it as test material unless we can assert its license and it is ALv2 compatible.  Its better to use entirely synthetic test data.

Reply:
Yes, it absolutely must be changed because the BBC is asserting its IP rights here:

```
<copyright><![CDATA[Copyright: (C) British Broadcasting Corporation, see http://news.bbc.co.uk/2/hi/help/rss/4498287.stm for terms and conditions of reuse.]]></copyright>
```